packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -162,6 +162,15 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     return (uint256(tobinTaxCache.numerator), TOBIN_TAX_DENOMINATOR);
   }
 
+  /**
+   * @notice Returns the tobin tax.
+   * @return The tobin tax amount as a fraction.
+   */
+  function getTobinTax() external view returns (uint256, uint256)
```

Comment:
Should this check to see if the tobin tax cache is stale? Otherwise you could run into a situation where it is, and when refreshed upon tx send it will be non-zero, but we return zero here, leading to a difference between what we show the user and reality.

Reply:
Good point. Renamed to `getPossiblyStaleTobinTax()` and it also now returns a boolean indicating whether it's stale or not.

Reply:
Why not just calculate the current tobin tax if the cache is stale, and return that?

Reply:
Switched to this, didn't realize it was possible as I had assumed `computeTobinTax()` would require a transaction to pay for computation (not the case as it doesn't change state). This now just looks like a non-state-changing version of `getOrComputeTobinTax()`. Thank you for the patience as I learn Solidity :-)